orga/app/components/campaign/update-form.js

```diff
@@ -38,20 +38,17 @@ export default class UpdateForm extends Component {
 
   @action
   onChangeCampaignName(event) {
-    const nameTrim = event.target.value.trim();
-    this.args.campaign.name = nameTrim || null;
+    this.args.campaign.name = event.target.value || null;
```

Comment:
le || null est géré par le nullableString non ?
```suggestion
    this.args.campaign.name = event.target.value;
```